liquibase-core/src/main/java/liquibase/snapshot/SnapshotControl.java

```diff
@@ -16,6 +16,14 @@ public class SnapshotControl implements LiquibaseSerializable {
     private Set<Class<? extends DatabaseObject>> types;
     private SnapshotListener snapshotListener;
 
+    private Set<Class<? extends DatabaseObject>> excludedTypes;
+
+    public SnapshotControl(Database database, List<String> excludeList) {
```

Comment:
Please move this constructor below the original one

Reply:
Aplied changes